### PR TITLE
fix: memory leak during inline webworker creation #208

### PR DESCRIPTION
**Pull request by michal-lipski**
This PR contains a:

- [x] **bugfix**
- [ ] new **feature**
- [ ] **code refactor**
- [x] **test update** 
- [ ] **typo fix**
- [ ] **metadata update**

### Motivation / Use-Case

solving memory issue described in #208 

### Breaking Changes

none

### Additional Info

* updated lint config to allow usage of global window object in tests
* updated jest config to mock global window object

### CODE REVIEW

**Comment by michal-lipski** [comment]
Hi @evilebottnawi I have added test that covers added code.

File: src/workers/InlineWorker.js [hunk 1]
```diff
@@ -1,7 +1,5 @@
 // http://stackoverflow.com/questions/10343913/how-to-create-a-web-worker-from-a-string
 
-var URL = window.URL || window.webkitURL;
```
Comment (evilebottnawi) [inline src/workers/InlineWorker.js]: Don't remove this, it is breaking change

Comment (evilebottnawi) [reply src/workers/InlineWorker.js]: My mistake

**Comment by ritz078** [comment]
any update on this ?

**Comment by michal-lipski** [comment]
Is there anything I can help with to merge this PR?

**Pull request closed without merging**